polkadot-parachains/src/chain_spec.rs

```diff
@@ -834,3 +834,252 @@ fn westmint_genesis(
 		},
 	}
 }
+
+/// We use the Kanvas runtime for Rokanvas hence the chain spec.
+pub type KanvasChainSpec = sc_service::GenericChainSpec<kanvas_runtime::GenesisConfig, Extensions>;
+
+/// Naming it after the polkadot parachain because the id is the same over all relay chains.
+const CANVAS_PARACHAIN_ID: u32 = 1002;
+
+/// The existential deposit is determined by the runtime "kanvas".
+const KANVAS_ED: kanvas_runtime::Balance = kanvas_runtime::constants::currency::EXISTENTIAL_DEPOSIT;
+
+pub fn rocanvas_development_config() -> KanvasChainSpec {
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenSymbol".into(), "ROC".into());
+	properties.insert("tokenDecimals".into(), 12.into());
+
+	KanvasChainSpec::from_genesis(
+		// Name
+		"Rokanvas Development",
```

Comment:
This is gonna sound radical, but I think it would be much less confusing if this were simply `Canvas Development`.

Same with the `Canvas Local` somewhere down below.

We already use the Canvas name for the existing Rococo deployment anyways.

Reply:
Either we give every parachain (i.e chain spec) a different name or we don't. But having `Kanvas` as a name for the Kusama parachain but naming the Rococo parachain `Canvas on Rococo` is inconsistent.